probe/main.go

```diff
@@ -144,8 +144,18 @@ func hostTopology(hostID, hostName string) report.Topology {
 	return t
 }
 
-func interrupt() chan os.Signal {
-	c := make(chan os.Signal)
-	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
-	return c
+func interrupt() {
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGQUIT)
+	buf := make([]byte, 1<<20)
+	for {
+		switch <-sigs {
+		case syscall.SIGINT:
+			log.Printf("=== received SIGINT ===\n*** exiting\n")
+			return
+		case syscall.SIGQUIT:
+			stacklen := runtime.Stack(buf, true)
+			log.Printf("=== received SIGQUIT ===\n*** goroutine dump...\n%s\n*** end\n", buf[:stacklen])
+		}
+	}
```

Comment:
D'oh.  Weave has this everywhere, so I assumed not.  Might be worth filing a bug with weave too.